post-install.md

```diff
@@ -47,3 +47,10 @@ WINEESYNC=1 WINEFSYNC=1 protontricks-launch --appid 489830 "$HOME/.config/modorg
 
 You can find the proper appid for the game you want [here](gamesinfo).
 
+## Pass argument to Mod Organizer 2
+
+The first launch option in game properties in Steam is passed as an argument to Mod Organizer 2. This allows to e.g. tell MO2 to launch an executable without having to open its UI and click the run button, like the shortcuts created by MO2 do on Windows.
+
+For example if you want MO2 to run an executable called `exe` directly, add the `moshortcut://exe` launch option in the Steam's game properties.
```

Comment:
I tend to opt for step-by-step instructions over descriptions, since they work a little better for users with almost no familiarity with the software they're using.
I'd also suggest adding an important note on the proper formatting of the launch option:

 ```suggestion
1. Open Steam;
2. Right click the game you want to launch directly and click on "Properties";
3. Scroll down to "Launch Options" within the "General" tab;
4. Write `'moshortcut://"executable name"'` in the launch options textbox. eg.:
	- `'moshortcut://"Fallout Launcher"'` will launch the original Fallout New Vegas launcher
	- `'moshortcut://"SKSE"'` will directly launch Skyrim and Skyrim SE, with SKSE enabled
5. Close the properties window and launch the game;

**IMPORTANT:** pay attention to the usage of single and double quotes in the examples above, as they ensure executable names including spaces will still work. The entire launch option should be wrapped in single quotes and the executable name should be wrapped in double quotes.
```